src/createServer.js

```diff
@@ -1,3 +1,66 @@
 // Write code here
 // Also, you can create additional files in the src folder
 // and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+
+function createServer() {
+  return http.createServer((req, res) => {
+    const url = new URL(req.url, `http://${req.headers.host}`);
+
+    const textToConvert = url.pathname.slice(1);
+
+    const params = Object.fromEntries(url.searchParams.entries());
+
+    const toCase = params.toCase;
+
+    const allCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+    const errors = [];
+
+    res.setHeader('Content-Type', 'application/json');
+
+    if (!textToConvert) {
+      errors.push({
+        message: 'Text to convert is required. Correct request is: '
+          + '"/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+      });
+    };
+
+    if (!toCase) {
+      errors.push({
+        message: '"toCase" query param is required. '
+        + 'Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+      });
+    };
+
+    if (!allCases.includes(toCase) && toCase) {
```

Comment:
trow differet errors. From the task:
![image](https://user-images.githubusercontent.com/99652389/196670097-7c483ca9-7365-4f10-b25b-1883b9141818.png)